LAS/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Linq;
+using WellLog.Lib;
+using WellLog.Lib.DataAccess;
+using WellLog.Lib.Validators;
+
+namespace LAS
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //setup our DI
+            var serviceCollection = new ServiceCollection();
+            WellLogModule.RegisterTypes(serviceCollection);
+
+            var serviceProvider = serviceCollection.BuildServiceProvider();
+
+            var lasLogFileDataAccess = serviceProvider.GetService<ILasLogFileDataAccess>();
+            var lasLog = lasLogFileDataAccess.Read("C:\\temp\\49009223560000.las");
+            foreach (var section in lasLog.Sections)
+            {
+                Console.WriteLine($"{section.SectionType}");
+
+                if (section.MnemonicsLines != null)
+                {
+                    foreach (var mnemonicLine in section.MnemonicsLines)
+                    {
+                        Console.WriteLine($"\tMnemonic: {mnemonicLine.Mnemonic}; Units: {mnemonicLine.Units}; Data: {mnemonicLine.Data}; Description: {mnemonicLine.Description}");
+                    }
+                }
+
+                if (section.AsciiLogDataLines != null)
+                {
+                    foreach (var asciiLine in section.AsciiLogDataLines.Take(25))
+                    {
+                        Console.WriteLine($"\t{string.Join('\t', asciiLine.Values)}");
+                    }
+                }
+            }
+
+            Console.WriteLine();
+            Console.WriteLine("----------");
+            Console.WriteLine();
+
+            var lasLogValidator = serviceProvider.GetService<ILasLogValidator>();
+            var validationErrors = lasLogValidator.ValidateLasLog(lasLog);
+            if (validationErrors.Any())
+            {
+                foreach (var verr in validationErrors)
+                {
+                    Console.WriteLine(verr.Message);
+                }
+            }
+            else
+            {
+                Console.WriteLine("No validation errors.");
+            }
```

Comment:
nit: Early return and save yourself from indentation to increase readability. Also move this to the same printer class above for displaying data to your user. You can then pass in an interface to it for how it's going to be printed and spy on that method for ensuring the correct data is outputted in a test.

```
if (!validationErrors.Any())
{
    Console.WriteLine("No validation errors.");
    return;
}

/// ... code for printing validation errors. 
```

Reply:
done in new pull request